feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/section/TimetableList.kt

```diff
@@ -94,9 +96,13 @@ fun TimetableList(
                             text = timetableItem.startsTimeString,
                             fontWeight = FontWeight.Medium,
                         )
-                        Text(text = "|")
+                        Text(text = "|",
+                            color = md_theme_light_outlineVariant,
+                        )
                         Text(
                             text = timetableItem.endsTimeString,
+                            color = md_theme_light_secondary,
```

Comment:
I think we should use a Theme color for this because it will break when you use dark theme 👀

Reply:
I fix it!